litestar/handlers/websocket_handlers/route_handler.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Mapping
+
+from litestar.exceptions import ImproperlyConfiguredException
+from litestar.handlers import BaseRouteHandler
+from litestar.types import Dependencies, Empty, EmptyType, ExceptionHandler, Guard, Middleware
+from litestar.types.builtin_types import NoneType
+from litestar.utils import is_async_callable
+
+if TYPE_CHECKING:
+    from litestar.dto.interface import DTOInterface
+
+
+class WebsocketRouteHandler(BaseRouteHandler["WebsocketRouteHandler"]):
+    """Websocket route handler decorator.
+
+    Use this decorator to decorate websocket handler functions.
+    """
+
+    __slots__ = ()
+
+    def __init__(
+        self,
+        path: str | None | list[str] | None = None,
+        *,
+        dependencies: Dependencies | None = None,
+        dto: type[DTOInterface] | None | EmptyType = Empty,
```

Comment:
Same comment as above: Because DTOs don't do anything here, they shouldn't be defined as kwargs.